src/components/tx-flow/common/TxNonce/index.tsx

```diff
@@ -82,17 +82,10 @@ const NonceFormOption = memo(function NonceFormOption({
   )
 })
 
-const getFieldMinWidth = (value: string, showRecommendedNonceButton = false): string => {
-  const MIN_CHARS = 5
+const getFieldMinWidth = (value: string): string => {
+  const MIN_CHARS = 7
   const MAX_WIDTH = '200px'
-  const ADORNMENT_PADDING = '24px'
-
   const clamped = `clamp(calc(${MIN_CHARS}ch + 6px), calc(${Math.max(MIN_CHARS, value.length)}ch + 6px), ${MAX_WIDTH})`
-
-  if (showRecommendedNonceButton) {
-    return `calc(${clamped} + ${ADORNMENT_PADDING})`
-  }
-
```

Comment:
Removed this to avoid the jumping when the reset button appears, it was really annoying. I made it wider by default.